frontend/src/api/network/servingRuntimes.ts

```diff
@@ -11,80 +11,66 @@ import { ServingRuntimeKind } from '../../k8sTypes';
 import { CreatingServingRuntimeObject } from 'pages/modelServing/screens/types';
 import { getModelServingRuntimeName } from 'pages/modelServing/utils';
 import { getModelServingProjects } from './projects';
+import { getConfigMap } from './configMaps';
+import { DEFAULT_MODEL_SERVING_TEAMPLATE } from 'pages/modelServing/screens/const';
+import YAML from 'yaml';
+
+const fetchServingRuntime = (
+  data: CreatingServingRuntimeObject,
+  configNamespace: string,
+  namespace: string,
+): Promise<ServingRuntimeKind> => {
+  return getConfigMap(configNamespace, 'servingruntimes-config')
+    .then((configmap) => {
+      const servingRuntime = YAML.parse(configmap.data?.['default-config'] || '');
```

Comment:
If somehow you failed to have a default-config, would we not want to fallback on `DEFAULT_MODEL_SERVING_TEAMPLATE`?

```
YAML.parse('');
// null
```

Reply:
Yes, it was already getting to the `DEFAULT_MODEL_SERVING_TEAMPLATE ` but for the wrong reasons, I'm throwing an error to just trigger the fallback.